lib/my_home_page.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+import 'package:the_movie_db_module2_part1/app_header.dart';
+import 'package:the_movie_db_module2_part1/related_genres_title.dart';
+import 'package:the_movie_db_module2_part1/genres_list.dart';
+import 'package:the_movie_db_module2_part1/backdrop.dart';
+import 'package:the_movie_db_module2_part1/movie_description.dart';
+import 'package:the_movie_db_module2_part1/movie_info.dart';
+
+class MyHomePage extends StatefulWidget {
+  const MyHomePage({super.key});
+
+  @override
+  State<MyHomePage> createState() => _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  Widget header = AppHeader();
+  Widget backdrop = BackDrop();
+  Widget movieInfo = const MovieInfo();
+  Widget relatedGenresTitle = RelatedGenresTitle();
+  Widget genresList = GenresList();
+  Widget movieDescription = MovieDescription();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        child: ListView(
+          children: [
+            header,
+            backdrop,
+            relatedGenresTitle,
+            genresList,
+            const SizedBox(height: 10),
+            movieInfo,
+            movieDescription,
+          ],
```

Comment:
You dont need to create a var to every Widget, you can create all directly on Listview children like:
```
ListView(
children: [
AppHeader(),
....
]
)
```